building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
You can quickly export your plugin into your running eclipse. It is useful if you don't want to run a build via cmd line. Your workspace already has TP set and eclipse can simply package it up and install it immediately. Why would they do this? If they were debugging a hard-to-debug issue that was only present in running builds (not runtime workbench) but didn't feel like switching to cmd line at all. 

This method is described as 'quick and dirrty'.